tools/custom-package-parser/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/pkg/file"
+	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func main() {
+	url := flag.String("url", "", "URL of the custom package")
```

Comment:
Feels like we could condense this to do starts-with http:// or https:// rather than needing to fiddle with flags. Non-blocking but I don't see a downside to simplifying here a bit? Could also split URL vs. path into separate functions to de-nest/simplify.

Reply:
Yeah, I didn't put a lot of thought into it, given it's a tool to gather data only.

Reply:
Fair enough. If I end up touching this as part of a subsequent ticket I'll clean it up then.